addons/payment_stripe/__manifest__.py

```diff
@@ -1,11 +1,12 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 {
-    'name': 'Stripe Payment Provider',
+    'name': 'Payment Provider: Stripe',
     'version': '2.0',
     'category': 'Accounting/Payment Providers',
-    'sequence': 395,
-    'summary': 'Payment Provider: Stripe Implementation',
+    'sequence': 350,
+    'summary': "Stripe is an Irish-American payment provider available on the US and supporting "
```

Comment:
I don't feel like you uniformized all module summaries. I don't think it needs to start with the name of the acquirer since it's written on the card. Please apply this comment to all other summaries.
![image](https://user-images.githubusercontent.com/6265134/193775663-8872132a-5582-429f-afb5-916b0134bdcc.png)

```suggestion
    'summary': "An Irish-American payment provider available in the US and supporting "
```